master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment        | Description                                | Schema | Default |
```

Comment:
Might be nice to make these consistent with the titles used for the calicoctl resources.

```
Field	Description	Accepted Values	Schema	Default
```